Sources/PackageGraph/PackageGraph.swift

```diff
@@ -227,6 +233,8 @@ extension PackageGraphError: CustomStringConvertible {
             return "multiple products named '\(product)' in: '\(packages.joined(separator: "', '"))'"
         case .multipleModuleAliases(let target, let product, let package, let aliases):
             return "multiple aliases: ['\(aliases.joined(separator: "', '"))'] found for target '\(target)' in product '\(product)' from package '\(package)'"
+        case .invalidSourcesForModuleAliasing(let target, let product, let package):
+            return "invalid sources for module aliasing; to enable it, target '\(target)' in product '\(product)' from package '\(package)' should only contain Swift source files"
```

Comment:
maybe this should say something like "module aliasing can only be used swift based targets"